core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -111,16 +111,17 @@ private[spark] class Benchmark(
     // The results are going to be processor specific so it is useful to include that.
     out.println(Benchmark.getJVMOSInfo())
     out.println(Benchmark.getProcessorName())
-    out.printf("%-40s %16s %12s %13s %10s\n", name + ":", "Best/Avg Time(ms)", "Rate(M/s)",
-      "Per Row(ns)", "Relative")
-    out.println("-" * 96)
+    out.printf("%-40s %16s %12s %13s %10s %13s\n", name + ":", "Best/Avg Time(ms)", "Rate(M/s)",
+      "Per Row(ns)", "Relative", "Stdev (ms)")
```

Comment:
Currently, this adds the new value at the end. Can we move this to `Best/Avg Time(ms)` group? For example, `Best/Avg/Stdev Time(ms)`?
```
Limiting:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative    Stdev (ms)
--------------------------------------------------------------------------------------------------
Top-level column                   231 /  240          4.3         230.7       1.0X            11
Nested column                     1833 / 1957          0.5        1833.1       0.1X            68
```

Reply:
I guess `Best/Avg/Stdev (ms)` will be enough because we use `Per Row(ns)` already.

Reply:
@dongjoon-hyun I thought about this. But then the readability of the numbers might be worse.
How about make each of them a single column? E.g. 
`Best Time(ms)    Avg Time(ms)    BestStdev Time(ms) `
I don't have a strong preference here.

Reply:
If we're going to add it, it doesn't make sense to do it separately at the end. I think best, avg, and stdev should be their own columns now.

Reply:
I got it, @srowen .

Reply:
Yup, I can separate it and place it after "avg" and before "rate"

Reply:
ok it looks like this now:
```
[info] agg w/o group:                            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------------------------------
[info] agg w/o group wholestage off                      43309          43591         283         48.4          20.7       1.0X
[info] agg w/o group wholestage on                        1032           1111         111       2032.4           0.5      42.0X
```